appveyor.yml

```diff
@@ -49,9 +55,9 @@ test_script:
   - "%CONDA% config --set always_yes true"
   - "%CONDA% create --name \"ibis_%PYTHON_VERSION%\" python=%PYTHON_VERSION% --channel conda-forge"
   - "%ACTIVATE% \"ibis_%PYTHON_VERSION%\""
-  - "%CONDA% install --channel conda-forge pytables numpy sqlalchemy psycopg2 graphviz click mock plumbum flake8 pytest"
+  - "%CONDA% install --channel conda-forge pytables numpy sqlalchemy psycopg2 pymapd graphviz click mock plumbum flake8 pytest"
   - "%CONDA% list"
-  - "pip install -e .\"[sqlite, postgres, mysql, visualization, pandas, csv, hdf5]\""
+  - "pip install -e .\"[sqlite, postgres, mysql, visualization, pandas, csv, hdf5, mapd]\""
```

Comment:
We probably do not want to test `mapd` on Windows since we don't have a way to hit a mapd database on CI without docker, and even if appveyor has docker and there's a mapd-for-windows docker image we should set that up in a follow up PR.

Reply:
ok!